### ISSUE TITLE
How to populate 'Link Text' popup field with the current selection when clicking 'link' button ?

### ISSUE THREAD

**Issue by ekaterinashenets** [nhn/tui.editor#2106]
## Summary
**Bold**, _italic_, etc. functions use text that is pre-selected for execution. 
I expect that the same to happen when selecting the text and clicking the `link` button to open the link creation popup. But the `Link Text` field is empty.
Is there any possibility to do this with the current implementation? 

## Screenshots
![image](https://user-images.githubusercontent.com/43377137/145576804-c9babdef-67e3-4889-a4be-4f13447d3b3f.png)

## Version
3.1.1

**Comment by js87zz** [comment]
@ekaterinashenets 
There is no way to solve this issue from the outside. 
I'll fix it in the next release. Thank you!

**Comment by ekaterinashenets** [comment]
Thanks for the quick response @js87zz !
What is the approximate date for the next release?

**Comment by cary-hu** [comment]
+1 for this, nowadays, I have tried 
```js
this._editor.setSelection(nodeInfo.range[0], nodeInfo.range[1]);
this._defaultText = this._editor.getSelectedText();
```
to get the link text and, 
```js
const Convertor = (this._editor as any).convertor;
const nodeAttrs = Convertor.focusedNode?.content?.content[0]?.marks[0]?.attrs;
if(!nodeAttrs) {
    return;
}
this._defaultLink = nodeAttrs.linkUrl;
```
to get the link url

**Comment by cary-hu** [comment]
There will got some error if link in list or table etc... so update:

```js
const targetNode = this.getLinkNodeAttrContent(Convertor.getInfoForPosSync()?.node);
const nodeAttrs = targetNode?.marks[0]?.attrs;
```
```js
private getLinkNodeAttrContent(node: any): any {
        if(node?.marks[0]?.type?.name === "link") {
            return node;
        }
        if(node?.content?.content?.length <= 0) {
            return;
        }
        for (let index = 0; index < node?.content?.content?.length; index++) {
            const element = node?.content?.content[index];
            const attr = this.getLinkNodeAttrContent(element);
            if(attr) {
                return attr;
            }
        }
    }
```

**Comment by js87zz** [comment]
@ekaterinashenets 
It will be fixed within this week.

@cary-hu 
I don't know exactly what you want.🧐 If your issue is same as this issue, it will be resolved soon, and if it is different, please write new issue.

**Comment by cary-hu** [comment]
hh same issue here, and this is a workaround so that I can get the link text and url and then change the exist one

**Comment by js87zz** [comment]
@ekaterinashenets 
The bug has been fixed in `v3.1.2`. Thanks!
